nsxt/resource_nsxt_policy_tier1_gateway_interface.go

```diff
@@ -42,6 +45,12 @@ func resourceNsxtPolicyTier1GatewayInterface() *schema.Resource {
 				Description: "Locale Service ID for this interface",
 				Computed:    true,
 			},
+			"site_path": {
```

Comment:
should this be ForceNew? Both in T1 and T0

Reply:
Thanks for the catch. Will fix for both t1 and t0.

> should this be ForceNew? Both in T1 and T0